client/containers/Pub/PubSubmissionHeader/spubHeader.scss

```diff
@@ -0,0 +1,43 @@
+.spub-header-component {
+	border-bottom:	border-bottom: 1px solid #d2d2d2;
+;
+	padding: 200px;
```

Comment:
Wait...what is this massive padding? 😆 I assume it's helpful for Storybook, but please remove it before landing.